sos/plugins/openstack_cinder.py

```diff
@@ -45,6 +47,9 @@ def setup(self):
         else:
             self.add_copy_spec_limit("/var/log/cinder/*.log",
                                      sizelimit=self.limit)
+        if self.get_option("exclude"):
+            exclude = self.get_option("exclude")
+            self.add_forbidden_path(exclude)
```

Comment:
> The file names can be different for different deployments.

Both the add and prohibit interfaces support globs; if you can't even be certain where the files are located then there are much bigger problems here.